controller_manager:
  ros__parameters:
    update_rate : 100
    joint_trajectory_controller:
      type: "joint_trajectory_controller/JointTrajectoryController"

    joint_state_broadcaster:
      type: "joint_state_broadcaster/JointStateBroadcaster"

joint_trajectory_controller:
  ros__parameters:
    joints:
      - shoulder_pan_joint
      - shoulder_lift_joint
      - elbow_joint
      - wrist_1_joint
      - wrist_2_joint
      - wrist_3_joint
      - plate_slider_joint
    command_interfaces:
      - position

    state_interfaces:
      - position
      - velocity

    state_publish_rate: 50.0
    action_monitor_rate: 20.0

    allow_partial_joints_goal: false
    open_loop_control: true
    constraints:
      stopped_velocity_tolerance: 0.01
      goal_time: 0.0
      joint1:
        trajectory: 0.05
        goal: 0.03